pkg/exchange/okex/parse.go

```diff
@@ -15,23 +15,56 @@ import (
 	"github.com/c9s/bbgo/pkg/types"
 )
 
-func parseWebSocketEvent(str []byte) (interface{}, error) {
-	v, err := fastjson.ParseBytes(str)
+func (s *Stream) parseWebSocketEvent(in []byte) (interface{}, error) {
+	var e WsEvent
+
+	err := json.Unmarshal(in, &e)
 	if err != nil {
 		return nil, err
 	}
+	switch {
+	case e.IsOp():
+		// need unmarshal again because arg in both WebSocketOpEvent and WebSocketPushDataEvent
+		var opEvent WebSocketOpEvent
+		err := json.Unmarshal(in, &opEvent)
+		if err != nil {
+			return nil, err
+		}
+		return &opEvent, nil
+	case e.IsPushDataEvent():
+		// need unmarshal again because arg in both WebSocketOpEvent and WebSocketPushDataEvent
+		var pushDataEvent WebSocketPushDataEvent
+		err := json.Unmarshal(in, &pushDataEvent)
+		if err != nil {
+			return nil, err
+		}
+		channel := pushDataEvent.Arg.Channel
 
-	if v.Exists("event") {
-		return parseEvent(v)
-	}
+		// keep old style first, just demo account channel, account channel use new writing style.
+		v, err := fastjson.ParseBytes(in)
+		if err != nil {
+			return nil, err
+		}
```

Comment:
I'm confused. Is there any reason to apply fastjson and json unmarshal at the same time?

Reply:
Originally I send this PR https://github.com/c9s/bbgo/pull/1321 and yes it's hard to read, so @bailantaotao suggest me to split more small PRs. So I just use new `unmarshal` way to `account` channel to demo what I want to do, others channel just keep their old  `fastjson` way, but I will change it in the future if this PR approved and merged.

Reply:
Yes, it's my idea. The original PR has too many comments, making it difficult to read and track each resolved item.